<ui version="4.0" >
 <class>SrcSelect</class>
 <widget class="QWidget" name="SrcSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="buttonGroup2" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="checkClipboard" >
        <property name="text" >
         <string>Clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="checkFile" >
          <property name="text" >
           <string>File:</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="newFilename" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="mode" >
           <enum>KFile::File|KFile::ExistingOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="checkServer" >
          <property name="text" >
           <string>Keyserver:</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="keyIds" >
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Enter the ids or fingerprints of the keys to import as space separated list.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>331</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >kcombobox.h</include>
 </includes>
 <resources/>
</ui>
